src/applications/edu-benefits/0994/pages/trainingProgramsInformation.js

```diff
@@ -20,6 +20,9 @@ const checkLocation = field => field === 'inPerson' || field === 'both';
 const showLocation = (formData, index) =>
   checkLocation(getCourseType(formData, index));
 
+const getProgramName = (formData, index) =>
```

Comment:
Is the only thing you're using this function for to test whether or not there's a program name? If that's all it's going to be used for, it might be clearer to name it something like `hasProgramName` and return a boolean.

Reply:
Thanks, I think the requirements changed again and we have another PR that'll be coming for the exact some thing =/